apps/src/sites/studio/pages/maker/discountcode.js

```diff
@@ -24,10 +25,20 @@ $(document).ready(() => {
   }
 
   ReactDOM.render(
-    <EligibilityChecklist
-      statusPD={scriptData.is_pd_eligible ? Status.SUCCEEDED : Status.FAILED}
-      statusStudentCount={scriptData.is_progress_eligible ? Status.SUCCEEDED : Status.FAILED}
-    />
+    <div>
+      {!(scriptData.is_pd_eligible && scriptData.is_progress_eligible) &&
+        <EligibilityChecklist
+          statusPD={scriptData.is_pd_eligible ? Status.SUCCEEDED : Status.FAILED}
+          statusStudentCount={scriptData.is_progress_eligible ? Status.SUCCEEDED : Status.FAILED}
+        />
+      }
+      {(scriptData.is_pd_eligible && scriptData.is_progress_eligible) &&
```

Comment:
I don't think this is the right check? When you begin the application process, both of those things will be true, but we still want to show you the EligibilityChecklist.

I think you probably want some new value (that the server is not yet sending) like `scriptData.discountCode`. If we have one, we can show the instructions, if we don't, we still need to work our way through the checklist.